src/BoundedInts.dfy

```diff
@@ -35,4 +33,26 @@ module BoundedInts {
   newtype nat32 = x: int  | 0 <= x < 0x8000_0000
   newtype nat64 = x: int  | 0 <= x < 0x8000_0000_0000_0000
 
+  const UINT8_MAX:  uint8  := 0xFF
+  const UINT16_MAX: uint16 := 0xFFFF
+  const UINT32_MAX: uint32 := 0xFFFF_FFFF
+  const UINT64_MAX: uint64 := 0xFFFF_FFFF_FFFF_FFFF
+
+  const INT8_MIN:  int8  := -0x80
+  const INT8_MAX:  int8  :=  0x7F
+  const INT16_MIN: int16 := -0x8000
+  const INT16_MAX: int16 :=  0x7FFF
+  const INT32_MIN: int32 := -0x8000_0000
+  const INT32_MAX: int32 :=  0x7FFFFFFF
+  const INT64_MIN: int64 := -0x8000_0000_0000_0000
+  const INT64_MAX: int64 :=  0x7FFFFFFF_FFFFFFFF
+
+  const NAT8_MAX:  nat8  := 0x7F
+  const NAT16_MAX: nat16 := 0x7FFF
+  const NAT32_MAX: nat32 := 0x7FFFFFFF
+  const NAT64_MAX: nat64 := 0x7FFFFFFF_FFFFFFFF
```

Comment:
There might be some redundancy with the PR with the Heap. Could you synchronize with @prvshah51 to ensure we define these constant in only one place?

Reply:
I think Parva didn't include these in the end; right?  So no conflict.